instrumentation/aws-java-sdk-kinesis-2.0.6/src/main/java/com/agent/instrumentation/awsjavasdk2/services/kinesis/StreamProcessedData.java

```diff
@@ -0,0 +1,19 @@
+package com.agent.instrumentation.awsjavasdk2.services.kinesis;
```

Comment:
In this case it makes sense to keep this class to be cached.
Because in this instrumentation, the streamName may be derived from the ARN.